bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/BreadcrumbImpl.java

```diff
@@ -36,6 +36,9 @@
 import com.day.cq.wcm.api.Page;
 import com.day.cq.wcm.api.designer.Style;
 
+import static com.adobe.cq.wcm.core.components.internal.models.v1.PageListItemImpl.PN_DISABLE_SHADOWING;
```

Comment:
It's good to not use static imports.
You could import your PageListItem and refer to the your variable by
PageListItem.PN_DISABLE_SHADOWING.

Reply:
Hi, thank you for the comment. You are right, this is not a good case for static import, I will update the request.